scripts/cnv_wdl/germline/cnv_germline_case_workflow.wdl

```diff
@@ -0,0 +1,355 @@
+            num_intervals_per_scatter = num_intervals_per_scatter,
+            gatk_docker = gatk_docker,
+            preemptible_attempts = preemptible_attempts
+    }
+
+    scatter (scatter_index in range(length(ScatterIntervals.scattered_interval_lists))) {
+        call GermlineCNVCallerCaseMode {
+            input:
+                scatter_index = scatter_index,
+                read_count_files = CollectCounts.counts,
+                contig_ploidy_calls_tar = DetermineGermlineContigPloidyCaseMode.contig_ploidy_calls_tar,
+                gcnv_model_tar = gcnv_model_tars[scatter_index],
+                intervals = ScatterIntervals.scattered_interval_lists[scatter_index],
+                gatk4_jar_override = gatk4_jar_override,
+                gatk_docker = gatk_docker,
+                mem = mem_for_germline_cnv_caller,
+                cpu = cpu_for_germline_cnv_caller,
+                p_alt = gcnv_p_alt,
+                cnv_coherence_length = gcnv_cnv_coherence_length,
+                max_copy_number = gcnv_max_copy_number,
+                mapping_error_rate = gcnv_mapping_error_rate,
+                sample_psi_scale = gcnv_sample_psi_scale,
+                depth_correction_tau = gcnv_depth_correction_tau,
+                copy_number_posterior_expectation_mode = gcnv_copy_number_posterior_expectation_mode,
+                active_class_padding_hybrid_mode = gcnv_active_class_padding_hybrid_mode,
+                learning_rate = gcnv_learning_rate,
+                adamax_beta_1 = gcnv_adamax_beta_1,
+                adamax_beta_2 = gcnv_adamax_beta_2,
+                log_emission_samples_per_round = gcnv_log_emission_samples_per_round,
+                log_emission_sampling_median_rel_error = gcnv_log_emission_sampling_median_rel_error,
+                log_emission_sampling_rounds = gcnv_log_emission_sampling_rounds,
+                max_advi_iter_first_epoch = gcnv_max_advi_iter_first_epoch,
+                max_advi_iter_subsequent_epochs = gcnv_max_advi_iter_subsequent_epochs,
+                min_training_epochs = gcnv_min_training_epochs,
+                max_training_epochs = gcnv_max_training_epochs,
+                initial_temperature = gcnv_initial_temperature,
+                num_thermal_epochs = gcnv_num_thermal_epochs,
+                convergence_snr_averaging_window = gcnv_convergence_snr_averaging_window,
+                convergence_snr_trigger_threshold = gcnv_convergence_snr_trigger_threshold,
+                convergence_snr_countdown_window = gcnv_convergence_snr_countdown_window,
+                max_calling_iters = gcnv_max_calling_iters,
+                caller_update_convergence_threshold = gcnv_caller_update_convergence_threshold,
+                caller_admixing_rate = gcnv_caller_admixing_rate,
+                disable_annealing = gcnv_disable_annealing,
+                preemptible_attempts = preemptible_attempts
+        }
+    }
```

Comment:
@samuelklee @asmirnov239 We can exclude intermediate .tar.gz outputs iff we include _everything_ properly in the (g)VCF output. Perhaps we can make this a mandate, assume that it will be so in the next maintenance release (if not for the release), and remove intermediate .tar.gz for now.

Reply:
I'll make an issue and leave this for now.